ChangeLog

```diff
@@ -1,3 +1,9 @@
+2016-06-15 Jae Chang <jhchang@lanl.gov>
+	
+	Github exercise.
+
+
```

Comment:
@jhchang-lanl Too many blank lines.  Please remove 2 extra blank lines to keep the formatting consistent with the rest of the file. (Edit locally, commit, push.)